api/controllers/UserController.js

```diff
@@ -283,7 +283,7 @@ module.exports = {
         user: req.user.name,
         type: "banUser",
         content: "Banned /u/" + req.params.username
-      });
+      }).exec(function () {});
```

Comment:
[Actually it works in 2.2](https://github.com/YaManicKill/flairhq/blob/version-2.2/api/controllers/UserController.js#L298). I think I got rid of it while refactoring stuff for 2.3.